docs/docs/references/connectors/README.mdx

```diff
@@ -0,0 +1,114 @@
+import ConnectorPlatformIntro from './fragments/_connector_platform_intro.mdx';
+import ConnectorStorageIntro from './fragments/_connector_storage_intro.mdx';
+import ConnectorTypeIntro from './fragments/_connector_type_intro.mdx';
+
+# 🔌 Connectors
+
+## Introduction
+
+### What is a connector?
+
+_Connectors_ play a critical role in Logto. With their help, Logto enables end-users to use passwordless registration or sign-in and the capabilities of signing in with social accounts. With the increasing popularity of websites and applications, passwordless and social sign-ins allow users to avoid managing numerous accounts and passwords.
+
+:hushed: Sounds fascinating, Uh? See [Configure connectors](../../recipes/configure-connectors/) if you want to set up an existing connector. If you can not find the connector you want to set up, you may develop those connectors by following the guides in [Create your connector](../../recipes/create-your-connector/).
+
+## Compositions
+
+_Connectors_ are stored both locally and remotely:
```

Comment:
Seems that the connector data is divided into two parts, one part is hardcoded in the code, and another part is stored in the DB.